code/modules/mob/mob_defines.dm

```diff
@@ -139,6 +139,9 @@
 	var/area/lastarea = null
 
 	var/digitalcamo = 0 // Can they be tracked by the AI?
+	var/digitalinvis = 0 //Are they ivisible to the AI?
+	var/image/digitaldisguise = ""  //what does the AI see instead of them?

```

Comment:
Nitpick, please don't initial an image as a string... that's just... well it's a type mismatch technically!
thankfully "" is treated as false (as is null, 0 etc.) but please just typecast it as 
```var/image/digitaldisguise = null```